ci-operator/step-registry/cucushift/installer/check/reboot-nodes/cucushift-installer-check-reboot-nodes-ref.yaml

```diff
@@ -0,0 +1,20 @@
+ref:
+  as: cucushift-installer-check-reboot-nodes
+  from_image:
+   namespace: ocp
+   name: "4.14"
+   tag: upi-installer
+  commands: cucushift-installer-check-reboot-nodes-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: SIZE_VARIANT
+    default: ""
+  - name: ENABLE_REBOOT_CHECK
```

Comment:
Once we move the reboot nodes step to test in job config level, we can remove this ENV.